src/plugins/orangehrmTimePlugin/Dao/ProjectActivityDao.php

```diff
@@ -202,8 +202,9 @@ public function getDuplicatedActivities(int $fromProjectId, int $toProjectId): a
         $duplicatedActivityNames = array_column($q->getQuery()->execute(), 'name');
 
         return $this->createQueryBuilder(ProjectActivity::class, 'activity')
-            ->andWhere($q->expr()->in('activity.name', $duplicatedActivityNames))
+            ->andWhere($q->expr()->in('activity.name', ':duplicatedActivities'))
```

Comment:
was this from the beginning of the Time module?

Reply:
no no mistake I made when rewriting for group by